mtags/src/main/scala-3/scala/scala/meta/internal/pc/SymbolPrinter.scala

```diff
@@ -1,14 +1,25 @@
 package scala.meta.internal.pc
 
-import dotty.tools.dotc.printing.PlainPrinter
+import dotty.tools.dotc.printing.RefinedPrinter
 import dotty.tools.dotc.core.Contexts.Context
 import dotty.tools.dotc.core.Symbols._
 import dotty.tools.dotc.core.Names._
+import dotty.tools.dotc.core.NameOps._
 import dotty.tools.dotc.core.Types._
 import dotty.tools.dotc.core.Flags
 import scala.language.implicitConversions
 
-class SymbolPrinter(implicit ctx: Context) extends PlainPrinter(ctx) {
+class SymbolPrinter(implicit ctx: Context) extends RefinedPrinter(ctx) {
+
+  private val defaultWidth = 1000
+
+  override def nameString(name: Name): String = {
+    name.stripModuleClassSuffix.toString()
+  }
+
+  def typeString(tpw: Type) = {
```

Comment:
I would add return types to public methods

Reply:
Good idea, added!